Code/Editor/CMakeLists.txt

```diff
@@ -249,38 +249,14 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
         RUNTIME_DEPENDENCIES
             Gem::LmbrCentral
     )
-    ly_add_googletest(
-        NAME Legacy::EditorLib.Tests
-    )
-
-    ly_add_target(
-        NAME EditorLib.Camera.Tests ${PAL_TRAIT_TEST_TARGET_TYPE}
-        NAMESPACE Legacy
-        FILES_CMAKE
-            Lib/Tests/Camera/editor_lib_camera_test_files.cmake
-        INCLUDE_DIRECTORIES
-            PRIVATE
-                .
-        BUILD_DEPENDENCIES
-            PRIVATE
-                AZ::AzCore
-                AZ::AzTest
-                AZ::AzToolsFramework
-                AZ::AzTestShared
-                Legacy::EditorLib
-                Gem::Camera.Editor
-                Gem::AtomToolsFramework.Static
-        RUNTIME_DEPENDENCIES
-            Legacy::EditorLib
-    )
-
     ly_add_source_properties(
         SOURCES Lib/Tests/Camera/test_EditorCamera.cpp
         PROPERTY COMPILE_DEFINITIONS 
         VALUES CAMERA_EDITOR_MODULE="$<TARGET_FILE_BASE_NAME:Camera.Editor>"
     )
 
     ly_add_googletest(
-        NAME Legacy::EditorLib.Camera.Tests
+        NAME Legacy::EditorLib.Tests
```

Comment:
Well good thing the Camera gem self activates in it's CMakeLists.txt
Otherwise there would be the possibility that the user could build the Editor TARGET and the `Camera.Editor` TARGET not build.

Reply:
will add it as a runtime dependency, will make it clearer

Reply:
I just want to clarify the Camera Gem has nothing to do with these tests at all, this is the Editor camera system which has nothing to do with the Camera Gem (it would be nice to have the Camera Gem use this new functionality in future, it's on the Viewport teams list 🙂)